packages/loot-core/src/platform/server/fs/index.ts

```
@@ -0,0 +1,79 @@
+export { join } from './path-join';
+
+export declare function init(): void;
+export type Init = typeof init;
+
+export declare function getDataDir(): string;
+export type GetDataDir = typeof getDataDir;
+
+export declare function _setDocumentDir(dir: string): string;
+export type _setDocumentDir = typeof _setDocumentDir;
```

Biome (1.9.4)

[error] Shouldn't redeclare '_setDocumentDir'. Consider to delete it or rename it.

'_setDocumentDir' is defined here:


(lint/suspicious/noRedeclare)